### PR TITLE
ack-eliciting not ACK-eliciting

### LINKED ISSUE #3068: ACK-eliciting vs ack-eliciting
The transport text wildly disagrees on case even in neighbouring paragraphs.
Recovery uses ack-eliciting consistently, except Ack-eliciting at paragraph start.

### PR DESCRIPTION
**Pull request by ianswett**
Fixes #3068

### CODE REVIEW

**Comment by janaiyengar** [comment]
We use ACK for ACK frame and 'acknowledgment' instead of 'ack'. I think this should be ACK-eliciting. I suppose if ack-eliciting were defined as a term that might make it ok to use... can you bring the term over from the recovery doc into the transport one?

**Comment by ianswett** [comment]
Definition of ack-eliciting added.

File: draft-ietf-quic-transport.md [hunk 1]
```diff
@@ -190,6 +190,11 @@ QUIC packet:
 : A complete processable unit of QUIC that can be encapsulated in a UDP
   datagram.  Multiple QUIC packets can be encapsulated in a single UDP datagram.
 
+Ack-eliciting Packet:
+
+: A QUIC packet that contain frames other than ACK and PADDING elicits an ACK
```
Comment (martinthomson) [inline draft-ietf-quic-transport.md]:
```suggestion
: A QUIC packet that contains frames other than ACK and PADDING. These cause a recipient
   to send an acknowledgment within the maximum ack delay.
```

File: draft-ietf-quic-transport.md [hunk 2]
```diff
@@ -3061,13 +3066,13 @@ of doing so.
 
 Packets containing only ACK frames are not congestion controlled, so there are
 limits on how frequently they can be sent.  An endpoint MUST NOT send more than
-one ACK-frame-only packet in response to receiving an ACK-eliciting packet (one
+one ACK-frame-only packet in response to receiving an ack-eliciting packet (one
 containing frames other than ACK and/or PADDING).  An endpoint MUST NOT send a
-packet containing only an ACK frame in response to a non-ACK-eliciting packet
+packet containing only an ACK frame in response to a non-acl-eliciting packet
```
Comment (camshaft) [inline draft-ietf-quic-transport.md]: s/acl/ack

File: draft-ietf-quic-transport.md [hunk 3]
```diff
@@ -190,6 +190,11 @@ QUIC packet:
 : A complete processable unit of QUIC that can be encapsulated in a UDP
   datagram.  Multiple QUIC packets can be encapsulated in a single UDP datagram.
 
+Ack-eliciting Packet:
+
+: A QUIC packet that contains frames other than ACK and PADDING. These cause a
+  recipient to send an acknowledgment within the maximum ack delay.
```
Comment (janaiyengar) [inline draft-ietf-quic-transport.md]:
```suggestion
  recipient to send an acknowledgment within its maximum ACK delay ({{transport-parameter-definitions}}).
```

File: draft-ietf-quic-transport.md [hunk 4]
```diff
@@ -190,6 +190,12 @@ QUIC packet:
 : A complete processable unit of QUIC that can be encapsulated in a UDP
   datagram.  Multiple QUIC packets can be encapsulated in a single UDP datagram.
 
+Ack-eliciting Packet:
+
+: A QUIC packet that contains frames other than ACK and PADDING. These cause a
+  recipient to send an acknowledgment within its maximum ACK delay
```
Comment (janaiyengar) [inline draft-ietf-quic-transport.md]:
```suggestion
  recipient to send an acknowledgment (see {{sending-acknowledgements}}).
```

File: draft-ietf-quic-transport.md [hunk 5]
```diff
@@ -190,6 +190,12 @@ QUIC packet:
...
+  ({{transport-parameter-definitions}}).
```
Comment (janaiyengar) [inline draft-ietf-quic-transport.md]:
```suggestion
```

**Pull request merged**